lib/importer/factory/object_factory.rb

```diff
@@ -25,12 +26,13 @@ def run
         object
       end
 
+      ## FOR CONSIDERATION: handle a row (i.e. Work) with more than one file:
+      ## currently the file_set is replaced on update
       def update
         raise "Object doesn't exist" unless object
-        run_callbacks(:save) do
```

Comment:
Why are all the callbacks removed?

Reply:
I didn't understand their use! Everything seems to work without. With them the method's Assignment Branch Condition size throws a rubocop error - would you prefer me to change the rubocop threshold for this too?

Reply:
Yes, adjust the thresholds as needed.  If the callbacks aren't used, then make a separate PR to   "remove unused callbacks"  It's easer to understand PRs if they constrain themselves to a single change.

Reply:
Ok thanks! I just restored the callbacks - that didn't cause a rubocop failure anymore. I think I was getting that on my initial solution before extracting logic in new methods.